index.html

```diff
@@ -304,9 +309,9 @@ <h3>Digital Integration Hub</h3>
                             through Apache Ignite’s functionally rich, unified and consistent APIs.
                         </p>
 
-                        <a href="/use-cases/digital-integration-hub.html" class="btn btn-primary" title="Digital Integration Hub"
+                        <a href="/use-cases/digital-integration-hub.html"  role="button" class="btn btn-primary" aria-label="Learn More About Digital Integration Hub" title="Digital Integration Hub"
                            onclick="ga('send', 'event', 'apache_ignite_use_cases', 'homepage_click', 'digital_hub');">
-                            Learn More</a>
+                           Learn More About Digital Integration Hub</a>
```

Comment:
Please revert this change, that's too much for the button title. I believe we can sacrifice this SEO optimization in favor of better readability. Digital Hub keyword is already mentioned a couple of times on this page.